### DISCUSSION TITLE
SVG image not rendered properly on using WkWebView instead of UIWebView

### DISCUSSION THREAD

**Discussion by YogaPriyaShanmugam** [xamarin/Xamarin.Forms#11905] [General]
I have used WkWebView instaed of UIWebView. I have used svg content but it is not located at the center of the frame. Right side of the image is cropped. How to make the content to be placed at the center.

          ` webView = new CustomWebView(new CGRect(0, 0, 150, 150), new WKWebViewConfiguration());
            var path = NSBundle.MainBundle.PathForResource(imageName, "svg");
            var url = new NSUrl(path);
            var urlreq = new NSUrlRequest(url);

            webView.ContentMode = UIViewContentMode.ScaleToFill;

            string index1 = path;
            if (System.IO.File.Exists(index1))
            {
                webView.LoadFileUrl(new NSUrl("file://" + index1), new NSUrl("file://" + index1));
            }
           
            return webView;

        public class CustomWebView : WKWebView
        {

            public CustomWebView(CGRect frame, WKWebViewConfiguration configuration) : base(frame, 
                 configuration)
            {
            BackgroundColor = UIColor.Clear;
            Opaque = false;
            UserInteractionEnabled = false;
           
            }
        }

Output Image
![InkedIMG_1185_LI](https://user-images.githubusercontent.com/20275526/91069326-e3b1ab80-e652-11ea-9c5a-59585ef546b0.jpg)
`

**Comment by EmilAlipiev** [comment]
you should create an issue not as discussion.
